test/unit/lazyAndCommands/lazy-devices-test.js

```diff
@@ -102,6 +122,21 @@ var iotAgentLib = require('../../../lib/fiware-iotagent-lib'),
         type: 'Motion',
         service: 'smartGondor',
         subservice: 'gardens'
+    },
+    device3 = {
+        id: 'TestRobotPre',
+        type: 'RobotPre',
+        service: 'smartGondor',
```

Comment:
Should be "smartgondor" ?

Reply:
Yes, we should follow the recommendation of using service names in lowercase. Nevertheless, in order to be consistent with the rest of the tests, I have maintained smartGondor.

You can check this in https://github.com/telefonicaid/iotagent-node-lib/blob/master/test/unit/lazyAndCommands/lazy-devices-test.js#L97, and https://github.com/telefonicaid/iotagent-node-lib/blob/master/test/unit/lazyAndCommands/lazy-devices-test.js#103. It is also the same in the rest of tests.

Reply:
Ok, let's use smartGondor to keep style. NTC